driver-api/src/main/java/io/openmessaging/benchmark/driver/BenchmarkProducer.java

```diff
@@ -13,6 +13,7 @@
  */
 package io.openmessaging.benchmark.driver;
 
+import java.io.Serializable;
```

Comment:
Can the serializer change be made without modifying this? One less thing we have to worry about trying to figure out how to merge back upstream.

Reply:
Ah that import line? That snuck in.